gremlinapi/__init__.py

```diff
@@ -42,7 +42,22 @@
 
 
 # Logging Configuration
-logging.getLogger('GremlinAPI.client').addHandler(logging.StreamHandler())
+class SecretsFilter(logging.Filter):
+    def filter(self, record):
```

Comment:
For what its worth, based on our security requirements, the minimum password length is 6 characters, and our API keys and bearer tokens will always be around 24 characters, so these if statements will always evaluate to true.

Reply:
That assumes correct values in place; what I'm looking for there is something that can generally be considered sane; e.g the user didn't populate one of those fields with a simple `.` which would effectively destroy the logs/debugging filter

Reply:
Ahh okay makes sense to me